docs/apidocs/index.rst

```diff
@@ -0,0 +1,12 @@
+.. _qiskit-qec:
+
+.. module:: qiskit_qec
```

Comment:
I _thought_ I had made this comment before, but apparently I didn't.

Does `qiskit[-_]qec` need to be renamed here?

Reply:
good catch!